lib/hls/hls_parser.js

```diff
@@ -2657,6 +2637,12 @@ shaka.hls.HlsParser = class {
       }
     }
 
+    if (contentType == shaka.util.ManifestParserUtils.ContentType.AUDIO) {
+      if (codecs == 'mp4a.40.34') {
```

Comment:
I think we should still link to https://bugs.chromium.org/p/chromium/issues/detail?id=489520 here.
Just in case Chromium decides to start supporting `mp4a.40.34`.

Reply:
Done!